shell/common/common_benchmarks.sh

```diff
@@ -377,6 +387,32 @@ get_initial_disk() {
   echo -e "$dir"
 }
 
+set_benchmark_data_sizes() {
+  BENCH_DATA_SIZE=$1
+  BENCH_DATA_SIZE_MB=$(( BENCH_DATA_SIZE / 1000000 ))
+  BENCH_DATA_SIZE_GB=$(( BENCH_DATA_SIZE / 1000000000 ))
+  BENCH_DATA_SIZE_TB=$(( BENCH_DATA_SIZE / 1000000000000 ))
+}
+
+get_bench_data_size_bytes() {
+    awk '
+    BEGIN{
+      s["TB"]=1000000000000
+      s["GB"]=1000000000
+      s["MB"]=1000000
+    }
```

Comment:
If we're definitely going to go as low as bytes, should we also include "KB" here? Otherwise, seems strange to jump from 1 to 1000000.

Reply:
The problem is that many benchmarks need the input size as bytes, that is why we need to save BENCH_DATA_SIZE in bytes. However our use cases include MB, GB and TB, it does not make sense to add KB.

Reply:
The only use cases for data size start at MB, we don't work with KB. In any case it is useful that the user can select the data size starting with MB up to TB, our current limit. 

We keep BENCH_DATA_SIZE in bytes as some benchmarks expect the size in bytes not GB or TB as in the case of Hadoop-Examples.